lib/submissions.ts

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import { Submission, StepAnswers } from 'data/flexibleForms/forms.types';
+
+const { ENDPOINT_API, AWS_KEY } = process.env;
+
+const headersWithKey = {
+  'x-api-key': AWS_KEY,
+};
+
+export const startSubmission = async (
+  formId: string,
+  socialCareId: number,
+  createdBy: string
+): Promise<Submission> => {
+  const { data } = await axios.post(
+    `${ENDPOINT_API}/submissions`,
+    {
+      formId,
+      socialCareId,
+      createdBy,
```

Comment:
at the moment this is just called `workerEmail`, but we want to somehow distinguish between editors and creators

Reply:
I think at creation (at least for now) it is fine to just submit createdBy.
The backend stores who created the submissions and also createdBy is assigned to a list of workers assigned to the submission. 

By default the person who create the submission is also assigned to work it, however, there is nothing stopping our patch endpoint to let us assign multiple workers to a submission or remove a worker from a submission.